src/pages/networks/forms/NetworkAcls.tsx

```diff
@@ -69,6 +71,27 @@ const NetworkAcls: FC<Props> = ({ formik, project }) => {
             id={networlAclSelectorId}
           />
         )}
+        {formik.values.readOnly ? (
+          formik.values.security_acls.length !== 0 && (
+            <NetworkDefaultACLReadOnly formik={formik} />
+          )
+        ) : (
+          <NetworkDefaultACLSelector
+            id={"NetworkDefaultACLSelector"}
+            isReadOnly={false}
+            onChange={(fieldValue, value) => {
+              formik.setFieldValue(fieldValue, value);
+            }}
+            values={{
+              Egress: formik.values.security_acls_default_egress ?? "",
+              Ingress: formik.values.security_acls_default_ingress ?? "",
+            }}
+            disabled={
+              !!formik.values.editRestriction ||
+              formik.values.security_acls.length === 0
+            }
```

Comment:
```suggestion
            disabled={formik.values.security_acls.length === 0}
```
We probably don't need the first line here, as this will only be rendered in edit mode enabled, which is blocked for users with restrictions.